### DISCUSSION TITLE
Need detailed explanation about the Settlement report in Amazon sp api.

### DISCUSSION THREAD

**Discussion by Karthick2299** [amzn/selling-partner-api-models#3744] [General]
Hi @chrisskhoury,

In the settlement report from the Amazon SP API, I'm able to retrieve the reportType `GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE`. However, I would like to understand the detailed explanation of the contents or attributes inside the settlement report like how the report is calculated or is there any formula to calculate it etc. Could anyone provide an explanation?

**Comment by nuvoretail-developer** [comment]
can you tell me how you retrived the data?

**Reply by Karthick2299** [reply]
SETUP POSTMAN

Firstly, In postman setup the Amazon sp api into it: 

Follow the steps from the link to integrate postman into Amazon sp api:
[POSTMAN SETUP](https://developer-docs.amazon.com/sp-api/docs/using-postman-for-selling-partner-api-models)

DOWNLOAD MODELS FROM AMAZON SP API.

Download the models from gitHub repository, it has json format, copy the data and then paste that into your local file and save that into your local machine.

[AMAZON SP API Models](https://github.com/amzn/selling-partner-api-models/tree/main/models)

Note: Add the models as per your wish.

Now, the Postman looks like this:
<img width="481" alt="Screenshot 2024-02-13 at 4 21 32 PM" src="https://github.com/amzn/selling-partner-api-docs/assets/106722215/a95d83a5-056f-425f-88ca-3b420c7f2a79">


DOWNLOAD SETTLEMENT REPORT USING POSTMAN:

- Generate the access token with valid seller Credentials.
- In Selling Partner API for Reports folder in postman, 
- Invoke GET reports API call to get all available reports.

Follow this link to get the SETTLEMENT reports
[Get Reports From Amazon SP API](https://developer-docs.amazon.com/sp-api/docs/reports-api-v2021-06-30-reference#getreport)

- Get the reportDocument Id from Response body of the Get reports response.
- And then, Invoke the Get ReportDocument Id API, then In response body .It has the download url. 
- Click that link and you’ll be able to download the report you want.
